src/configs/ToastConfig.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import {View, Text, StyleSheet} from 'react-native';
 import {COLORS} from '../utils/theme';
+import {fonts, normalizeFontSize} from '../utils/fonts';
+import dimen from '../utils/dimen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include `fonts` and `normalizeFontSize` from the `../utils/fonts` module, and `dimen` from the `../utils/dimen` module. This is a good practice as it allows for better consistency and maintainability of font sizes and dimensions across the application.

```diff
+ import {fonts, normalizeFontSize} from '../utils/fonts';
+ import dimen from '../utils/dimen';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->